spec/support/features/session_helper.rb

```diff
@@ -35,6 +35,16 @@ def sign_up
       user
     end
 
+    def sign_up_with_sp(loa: 'loa1')
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
+      user = sign_up_and_set_password
+      fill_in 'Phone', with: '202-555-1212'
+      send("#{loa}_sp_session")
+      select_sms_delivery
+
+      user
+    end
```

Comment:
How about moving this in to the `sign_up_spec`, since it is only used there, and change it slightly to pass in the loa level without having to use `send` and define a separate method:
```ruby
def begin_sign_up_with_sp_and_loa(loa3:)
  user = create(:user)
  login_as(user, scope: :user, run_callbacks: false)
  Warden.on_next_request do |proxy|
    session = proxy.env['rack.session']
    session[:sp] = { loa3: loa3, name: 'Your friendly Government Agency' }
  end
  visit profile_path
  fill_in 'Phone', with: '202-555-1212'
  select_sms_delivery
  user
end
```
Then, use it in the tests like this: `begin_sign_up_with_sp_and_loa(loa3: false)`